src/Symfony/Bundle/SecurityBundle/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 5.3
 ---
 
+ * Added the `debug:firewall` command.
```

Comment:
```suggestion
 * Add the `debug:firewall` command.
```

Reply:
The other lines in this file (<=5.2.0) use "Added" and "Removed". Or is this different because it's an upcoming version?

Reply:
We recently standarized the CHANGELOG conventions and it was decided to use imperative verbs: https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry & https://symfony.com/doc/current/contributing/code/pull_requests.html#work-on-your-pull-request

Reply:
Ah, I didn't know that. Thanks!